pkgs/development/libraries/phonon/backends/gstreamer.nix

```diff
@@ -7,14 +7,6 @@ stdenv.mkDerivation rec {
   pname = "phonon-backend-gstreamer";
   version = "4.10.0";
 
-  meta = with lib; {
-    homepage = "https://phonon.kde.org/";
-    description = "GStreamer backend for Phonon";
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ ttuegel ];
-    license = licenses.lgpl21;
-  };
-
```

Comment:
Oops? I don't think removing meta should be part of this commit.

Reply:
moved to #245102